### ISSUE TITLE
Cannot open vault offline - requires Internet connection

### ISSUE THREAD

**Issue by jeffshead** [bitwarden/android#3912]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

Open Bitwarden and use biometric fingerprint or PIN to login. After login and BW opens, I get the "Requires Internet" error.

### Expected Result

Open vault and work offline.

### Actual Result

Error stating Internet connection is required. Usually, I cannot get passed this error and have to force close BW and re-open.

### Screenshots or Videos
![1](https://github.com/user-attachments/assets/ecf5c9af-88ff-47f8-8f3d-f1acb2445eb0)

![2](https://github.com/user-attachments/assets/ba9e1a8d-2006-466f-a624-835ad82ad956)

![3](https://github.com/user-attachments/assets/b67dc168-4ec2-4df7-b515-628990e265db)

### Additional Context

Sometimes when I get the alert depicted in the first image above, clicking the OK button works. However, most of the time I just get the screen depicted in the second image and clicking on the "Try again" link doesn't help. Just keep getting that same screen with the "try again" link.  I have to force stop Bitwarden using the Android app settings/info menu. I can then re-open BW and use the the vault, offline. This has been an issue since the major update (stable release) and has plagued each subsequent beta release.

### Build Version

2024.8.1-beta (19099)

### Environment Details

Android 14
Pixel 4a5G

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hi there,

Are you doing that from a locked vault or fully logged out vault?

**Comment by jeffshead** [comment]
> Hi there,
> 
> Are you doing that from a locked vault or fully logged out vault?

Locked.

**Comment by Krychaz** [comment]
I notice that you are running the beta version of the app, is that correct?

Testing with the latest available version which is 2024.7.1 everything works as expected when accessing vault offline.

**Comment by jeffshead** [comment]
Yes, I'm using latest beta.

Do you know if v2024.7.1 suffers from the bio-metrics bug that's posted here: https://www.reddit.com/r/Bitwarden/comments/1erp2r1/comment/ll4goi0/?context=3

Also, I did not see a stable v2024.7.1 release; only v2024.7.1 beta.

**Comment by Krychaz** [comment]
2024.7.1 (11086) is the latest version available on Google Play Store 

This issue is still present in this version.

**Comment by jeffshead** [comment]
> 2024.7.1 (11086) is the latest version available on Google Play Store
> This issue is still present in this version.
FYI - The biometrics issue seems to be fixed in 2024.8.1 beta.

**Comment by Krychaz** [comment]
Thank you for confirming this. I have passed this issue to our engineering team to investigate ahead of release

**Comment by tomikaka22** [comment]
Not being able to access my passwords when offline has been a real issue for me multiple times now. Exact same error, exact same behaviour. 

I should add that I am self hosting a Vaultwarden instance and I can't reach it when I'm not at home, I am not truly offline.

**Comment by illspirit** [comment]
Yeah this has been awful to deal with. I'm running a LAN only server and I leave town and I can't access a local database?

**Comment by jeffshead** [comment]
When I opened this issue, I was on beta 2024.8.1. I've been using stable v2024.10.0 without experiencing this issue. So now it seems the issue has been reintroduced in later versions. Has anyone tried 2024.11.6 (19499)? I'm hesitant to try it because , for whatever reason, v2024.10.0 is not available for download from the releases page. It goes from 2024.9.0-beta to 2024.10.1? Why did they remove 2024.10.0?

**Comment by tomikaka22** [comment]
I remember the Vaultwarden github repo mentioning to report issues to them instead of bitwarden, maybe it's worth opening an issue there too.

**Comment by tomikaka22** [comment]
Using the latest version from F-Droid and I haven't had this issue in a while. Might be worth closing.

**Comment by jeffshead** [comment]
Haven't experienced this issue with more recent versions of BW. Closing this issue.
